samples/apps/copilot-chat-app/DocumentUploadApp/appsettings.json

```diff
@@ -0,0 +1,7 @@
+﻿{
+    "AppConfig": {
+        "ClientId": "0b4c2e65-3bfc-4bb3-8f98-5e8927dd98c8",
```

Comment:
Even though it's not a secret, if you created this Client ID, you probably don't want to check it in.